packages/lavapack/src/runtime.js

```diff
@@ -4408,27 +4420,64 @@ const loadWithoutErrorAnnotation=  async(
       } in compartment ${q(compartment.name)}`;
    }
 
-  if( staticModuleRecord.record!==  undefined) {
-    const {
-      compartment: aliasCompartment=  compartment,
-      specifier: aliasSpecifier=  moduleSpecifier,
-      record: aliasModuleRecord,
-      importMeta}=
-        staticModuleRecord;
+  // check if record is a RedirectStaticModuleInterface
+  if( staticModuleRecord.specifier!==  undefined) {
```

Comment:
The changes in this file seem unrelated to the CJS generation update of this PR? Can it be broken out as a separate change?

Reply:
I guess it comes from missing to regenerate after template changes in #435? If so it's better suited as an independent follow-up to that.

Reply:
#437

Reply:
3b7eb78